gno.land/pkg/gnoweb/gnoweb.go

```diff
@@ -330,6 +330,15 @@ func handleRealmRender(logger *slog.Logger, app gotuna.App, cfg *Config, w http.
 	queryParts := strings.Split(querystr, "/")
 	pathLinks := []pathLink{}
 	for i, part := range queryParts {
+		// Unescape query arguments when present in the last render path part
+		if i == len(queryParts)-1 {
+			part, err = unescapeQueryArgs(part)
+			if err != nil {
+				writeError(logger, w, err)
+				return
+			}
+		}
+
 		pathLinks = append(pathLinks, pathLink{
 			URL:  "/r/" + rlmname + ":" + strings.Join(queryParts[:i+1], "/"),
 			Text: part,
```

Comment:
Query arguments are unescaped when displayed in the breadcrumb.

Should query string be part of the breadcrumb? It increases the length of the breadcrumb which potentially might lead to breadcrumb wrapping.

Reply:
With the lates changes all URL query arguments are not displayed as part of the breadcrumb. They are available as part of the last breadcrumb item URL though.